remote/advancedcontainers/sharing-git-credentials.md

```diff
@@ -101,4 +105,5 @@ RUN sudo apt-get update && sudo apt-get install gnupg2 -y
 
 The next time the container starts, your GPG keys should be accessible inside the container as well.
```

Comment:
The container actually needs to be rebuilt for the changes to the Dockerfile to apply. Could you also update that, thanks!

Reply:
Updated the sentence.